### PR TITLE
Move Nelmio API doc to dev

### PR DESCRIPTION
**Pull request by vincentchalamon**

### CODE REVIEW

**Comment by dupuchba** [comment]
:+1:

**Comment by jderusse** [comment]
IMHO, documentation must be exposed publicly to consumers. It's not only a dev things.

An API is not only an endpoint, it's a documentation and SDK too

File: app/autoload.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 use Composer\Autoload\ClassLoader;
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
-/**
```
Comment (teohhanhui) [inline app/autoload.php]: This must be a mistake, right?

File: web/app_dev.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-/**
```
Comment (teohhanhui) [inline web/app_dev.php]: Why the change?

**Comment by teohhanhui** [comment]
I'm -1 for this.  Agree with @jderusse

File: web/app_dev.php (hunk 2 above)
Comment (theofidry) [inline web/app_dev.php]: in those cases (forcing a typehint with `@var` in inline code) phpdoc are unneeded, a simple comment do

Comment (teohhanhui) [inline web/app_dev.php]: I'd rather we don't change things unnecessarily from symfony-standard.

In any case, this syntax is actually still wrong anyway (according to PSR-5 draft). See relevant discussion here: https://www.drupal.org/node/2305593

Comment (theofidry) [inline web/app_dev.php]: I remember getting a warning from phpStorm, Scrutinizer or another tool
about that at some point which could explain why he changed that.

But in any case, thanks for looking into that anyway

Comment (vincentchalamon) [inline web/app_dev.php]: This is the recommendation from StyleCI, maybe @dunglas should adapt its configuration so

**Comment by vincentchalamon** [comment]
According to your comments, we keep NelmioAPIDoc as it. I'm closing this PR
